MobileWallet/Common/CopyableLabel.swift

```diff
@@ -130,7 +130,7 @@ extension CopyableLabel {
         greenView.backgroundColor = Theme.shared.colors.transactionCellValuePositiveText?.withAlphaComponent(0.12)
 
         let copiedLabel = UILabel()
-        copiedLabel.text = NSLocalizedString("COPIED", comment: "Emoji ID copied")
+        copiedLabel.text = NSLocalizedString("emoji-copied-action-title", comment: "Emoji ID copied")
```

Comment:
`emoji_id_copied` - it looks generic but if you have something similar in the future you can make it more specific. Shorter is better imo if it's possible to get away with it.